ldms/src/ldmsd/ldmsd.c

```diff
@@ -2097,13 +2126,6 @@ int main(int argc, char *argv[])
 		exit(0);
 	}
 
-	if (!foreground) {
-		if (daemon(1, 1)) {
-			perror("ldmsd: ");
-			cleanup(8, "daemon failed to start");
-		}
-	}
-
 	/*
 	 * TODO: It should be a better way to get this information.
```

Comment:
@nichamon can you elaborate on this comment?

Reply:
I added that when I refactored LDMSD to use the `ovis_log` library. The comment was to remind myself to find a better way for admins to change the time format of log messages from date-time to timestamp. Currently, admins could configure LDMSD to use timestamp from the epoch (seconds) in log messages by setting the environment variable `LDMSD_LOG_TIME_SEC`, which I didn't like this method. However, I didn't introduce an additional command-line option at the time because there were so many options already.

I don't remember the motivation of different time-formats of log messages. If I think of two path forwards:

1. We think there is no practical use of supporting multiple time formats in log messages.
In this case, I will remove the code block.

2. We decide to keep the support.
I would change the new `log_file` config command from
`log_file path=<log file path>`
to
`log dest=<log file path> [time_format=<date-time|timestamp|none>]`.
The `ovis_log` library supports the three modes: date-time (LDMSD default), timestamp in second from epoch, and no time information.
Beside modifying the config command, I would remove the logic to get the time format from the environment.

What do you think?